app/models/review_turnaround.rb

```diff
@@ -0,0 +1,31 @@
+# == Schema Information
+#
+# Table name: review_turnarounds
+#
+#  id                :bigint           not null, primary key
+#  value             :integer
+#  review_request_id :bigint           not null
+#
+# Indexes
+#
+#  index_review_turnarounds_on_review_request_id  (review_request_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (review_request_id => review_requests.id)
+#
+
+class ReviewTurnaround < ApplicationRecord
+  belongs_to :review_request
+
+  private
+
+  def build
+    self.value = calculate_turnaround
+    save!
+  end
+
+  def calculate_turnaround
+    review_request.reviews.first.opened_at.to_i - review_request.created_at.to_i
```

Comment:
so are you sure `review_request.reviews.first` will always return the oldest review? is the `order` specified somewhere?

Reply:
Perhaps this record should hold a reference to the `review`

Reply:
We have the condition in Review model where we ask if the review request has one review.